pages/give-feedback.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Head from 'next/head';
+
+import BackButton from "components/navigation/BackButton";
+import { site } from "constants/index"
+import SharedLayout from "components/SharedLayout";
+
+import Form from "components/forms/Form";
+import Input from "components/forms/Input";
+import Select from "components/forms/Select";
+import Checkbox from "components/forms/Checkbox";
+import FormLabel from "components/forms/FormLabel";
+import Submit from "components/forms/Submit";
+
+import TakeAction from "components/common/TakeAction";
+
+import { toolUseful } from 'constants/forms';
+import { learnFromTool } from 'constants/forms';
+
+export default function GiveFeedback() {
+
+    return (
+        <div>
+            <Head>
+                <title>Give Feedback - { site.name }</title>
+            </Head>
+            <SharedLayout>
+                <BackButton className="mt-3 mb-2" />
+                <h1>Give Feedback</h1>
+                <p>The Policy Map and Scorecard will only be effective if it is useful to our users. We want to hear from you how you used the tool and whether it was helpful. If you didn’t find the tool helpful, feel free to share how we can make the necessary improvements. Thanks so much for sharing your feedback and for helping us make the Policy Map and Scorecard the best tool it can possibly be!</p>
+                <Form
+                    className="col-12 col-lg-8 mb-5"
+                    path="give-feedback"
+                    form={props => (
+                        <form>
```

Comment:
This might benefit from some bootstrap classes for better layout on wide screens (I stole this from `partner-with-freefrom.js`:
```suggestion
                        <form className="col-12 col-lg-8 mb-5">
```

Reply:
Yes! I just made this change in the last commit (right before you reviewed)